app/src/main/java/org/simple/clinic/sync/DataSync.kt

```diff
@@ -66,20 +66,18 @@ class DataSync(
   private fun allSyncs(): Single<List<SyncResult>> {
     val syncAllGroups = SyncGroup
         .values()
-        .map(::syncsForGroup)
+        .map { syncsForGroup() }
 
     return Single
         .merge(syncAllGroups)
         .reduce(listOf(), { list, results -> list + results })
   }
 
-  private fun syncsForGroup(syncGroup: SyncGroup): Single<List<SyncResult>> {
-    val syncsInGroup = modelSyncs.filter { it.syncConfig().syncGroup == syncGroup }
-
+  private fun syncsForGroup(): Single<List<SyncResult>> {
     return Single
         .fromCallable { userSession.loggedInUserImmediate().toOptional() }
         .subscribeOn(schedulersProvider.io())
-        .compose(filterSyncsThatRequireAuthentication(syncsInGroup))
+        .compose(filterSyncsThatRequireAuthentication(modelSyncs))
```

Comment:
the `filterSyncsThatRequireAuthentication` function parameter name is not changed in the body. Can you update?

Reply:
Good catch, fixing this.